src/main/java/teammates/storage/api/FeedbackResponseMonitorDb.java

```diff
@@ -45,7 +45,7 @@ FeedbackResponseRecordAttributes makeAttributes(FeedbackResponseRecord entity) {
      * Gets a set of response records with chosen duration and interval.
      */
     public List<FeedbackResponseRecordAttributes> getResponseRecords(long duration, long interval) {
-        long startTimeInMilisec = System.currentTimeMillis() - duration;
+        long startTimeInMilisec = System.currentTimeMillis() - duration * 1000;
```

Comment:
I thought the front end would send milliseconds, why need to multiply?

Reply:
will change to milliseconds, was in seconds